tensorrt_llm/_torch/pyexecutor/py_executor_creator.py

```diff
@@ -220,7 +220,9 @@ def create_py_executor(executor_config: ExecutorConfig,
                 pytorch_backend_config,
                 batch_size=executor_config.max_batch_size,
                 max_num_tokens=executor_config.max_num_tokens,
-                max_seq_len=model_engine.max_seq_len,
+                # Note: The draft model engine will infer its own max_seq_len.
+                # We'll stop drafting when we hit the max.
+                max_seq_len=executor_config.max_seq_len,
```

Comment:
Is `executor_config.max_seq_len` different from `model_engine.max_seq_len`? What are the definitions for them?

Reply:
Yes. `executor_config.max_seq_len` is a user-specified maximum sequence length that is allowed to be smaller than the model's max seq len.